runtime/doc/api-functions.txt

```diff
@@ -0,0 +1,722 @@
+*api-funcs.txt*         Neovim API Function Reference                   {Nvim}
```

Comment:
Oops.  The script mostly had hard coded paths.  I turned them into variables so the script could be tweaked a bit in the future.  Guess I missed one.